### PR TITLE
Repeat arg fixes

### LINKED ISSUE #9: Repeat argument fixes
We need to:

1. Remove the nested repeat argument since it's [not supported in the AMPCS command dictionary spec](https://github.com/NASA-AMMOS/ampcs-dict-schemas/blob/a7b7c4e2436df8c319218cbf0f2741e8752a1785/CommandDictionary.rnc#L543):
https://github.com/NASA-AMMOS/seq-json-schema/blob/b56782b6434b3deb9fa60ddfec70cfc59877acec/schema.json#L445
2. Make the repeat arg an array of arrays so we can actually repeat the arguments.
https://github.com/NASA-AMMOS/seq-json-schema/blob/b56782b6434b3deb9fa60ddfec70cfc59877acec/schema.json#L424

We should add tests that cover both of these changes.

### PR DESCRIPTION
**Pull request by cohansen**
Closes #9.

I updated the repeat arg description as well to remove the nested information.
I also added an invalid test for nested repeat args.

### CODE REVIEW

**Review by camargo — APPROVED** [review]
LGTM, let's get sign off from Taifun and we can merge

File: schema.json [hunk 1]
```diff
@@ -422,27 +422,31 @@
           "const": "repeat"
         },
         "value": {
-          "description": "A repeat argument value, there can be a nested repeat arguments inside.",
+          "description": "A repeat argument value.",
           "type": "array",
           "items": {
             "oneOf": [
```
Comment (camargo) [inline schema.json]: Do we need this outer `oneOf`?

Comment (cohansen) [reply schema.json]: Yeah that's the outer array for the repeat.

Comment (camargo) [reply schema.json]: Can it just be:
```json
{
  "description": "A repeat argument value.",
  "type": "array",
  "items": {
    "type": "array",
    "items": {
      "oneOf": [
        {
          "$ref": "#/$defs/string_argument"
        },
        {
          "$ref": "#/$defs/number_argument"
        },
        {
          "$ref": "#/$defs/boolean_argument"
        },
        {
          "$ref": "#/$defs/symbol_argument"
        },
        {
          "$ref": "#/$defs/hex_argument"
        }
      ]
    }
  }
}
```
Just wondering if the outer array needs the `oneOf` or if it works without?

Comment (cohansen) [reply schema.json]: It looks like that's invalid because `items` needs to be an object not an array.

**Pull request merged**